src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_FunctionPointerInvocation.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
```

Comment:
>using [](start = 0, length = 5)

```#nullable enable``` ?

Reply:
Won't be relevant in next revision.

---
In reply to: [382382617](https://github.com/dotnet/roslyn/pull/41794#discussion_r382382617) [](ancestors = 382382617)